controllers/replicationdestination.go

```diff
@@ -0,0 +1,92 @@
+package controllers
+
+import (
+	"fmt"
+
+	volsyncv1alpha1 "github.com/backube/volsync/api/v1alpha1"
+	"github.com/go-logr/logr"
+	pvcv1alpha1 "github.com/konveyor/volume-snapshot-mover/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *DataMoverRestoreReconciler) CreateReplicationDestination(log logr.Logger) (bool, error) {
+
+	// get datamoverrestore from cluster
+	dmr := pvcv1alpha1.DataMoverRestore{}
+	if err := r.Get(r.Context, r.NamespacedName, &dmr); err != nil {
+		r.Log.Error(err, "unable to fetch DataMoverRestore CR")
+		return false, err
+	}
+
+	// get datamoverbackup from cluster
+	// TODO: name
+	dmb := pvcv1alpha1.DataMoverBackup{}
```

Comment:
Are we restoring DMB CR? My understanding is that we create a DMR CR (with all the values from DMB CR) for every DMB CR that we encounter in velero backup and ignore restoring DMB CR. Please correct me if I am missing something

Reply:
Just going to leave this in for now until we are able to get DMB from backup.